python/tests/aiconfigs/tarvel_gpt_prompts_with_dependency.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
nit: `tarvel_gpt_prompts_with_dependency.json` -> `travel_gpt_prompts_with_dependency.json`

Reply:
🤣 Nice find!

Reply:
What is this new example showing?

Reply:
It's a aiconfig which uses output from a previous prompt as input to another prompt.